src/transforms/aggregate.rs

```diff
@@ -0,0 +1,440 @@
+// let g:cargo_makeprg_params = = '--lib --no-default-features --features=transforms-aggregate transforms::aggregate'
```

Comment:
Can you drop this?

Reply:
Doh. That was just a note to myself when I was poling around and wasn't intended to make it into a commit.